src/hash.cpp

```diff
@@ -16,13 +16,26 @@
 
 #include <iostream>
 #include <string>
+#include <SDL_platform.h>
+
+// Encrypotion in iOS app complicates its registration.
+#if !defined(__IPHONEOS__)
 
 #include <openssl/sha.h>
 #include <openssl/md5.h>
 
 static_assert(utils::md5::DIGEST_SIZE == MD5_DIGEST_LENGTH, "Constants mismatch");
 static_assert(utils::sha1::DIGEST_SIZE == SHA_DIGEST_LENGTH, "Constants mismatch");
 
+#else
+
+#import <CommonCrypto/CommonDigest.h>
```

Comment:
This should probably be `#include` since this is a cpp file. (Don't they do basically the same thing anyway?)